packages/nimble-components/src/table/specs/table-column-specs/table-column-duration-text-hld.md

```diff
@@ -0,0 +1,200 @@
+
+### Risks and Challenges
+
+Formatting the duration units in the exact way that SLE presents them would require custom logic that differs from what the browser provides natively using APIs like `Intl.Numberformat.format()`. The preference here is to align with current browser behaviors for unit formatting, which includes localization.
+
+### Prior Art/Examples
+
+![ElapsedTime SLE](../spec-images/ElapsedTimeSLE.PNG)
+
+## Design
+
+Below is an example of how the `nimble-table-column-duration-text` would be used within a `nimble-table`:
+
+```HTML
+<nimble-table>
+    <nimble-table-column-duration-text field-name="duration">Duration</nimble-table-column-duration-text>
+</nimble-table>
+```
+
+The displayed value for duration will always take the form of "\<days display>, \<hour display>, \<minute display>, \<second display>.\<fractional seconds>", where the commas are non-localized delimiters <sup>1</sup>.
+
+<sup>1.</sup> When and if we are able to adopt the `Intl.DurationFormat` to format values, we will no longer be hard-coding the commas in the string result as that object will produce the entirety of the display text, which may not match what we produce in our implementation.
+
+### API
+
+#### `TableColumnDurationText`
+
+The `TableColumnDurationText` implementation requires no additional API from what `TableColumnTextBase` provides:
+
+```ts
+// inherits field-name attribute
+public class TableColumnDurationText : TableColumnTextBase { }
+
+```
+
+This API would result in a column whose value would only show up to the 'days' unit (i.e. it would show days, hours, minutes, and seconds). We would show up to two fractional digits as needed. Formatting to change slightly from what SLE offers now using "hr" instead of "h", and "sec" instead of "s".
+
+Examples:
+
+| English | German | French | Chinese |
+| ------- | ------ | ------ | ------- |
+| "1 hr, 30 min, 2 sec" | "1 Std., 30 Min., 2 Sek." | "1 h, 30 min, 2 s" | "1小时, 30分钟, 2秒" |
+| "2 hr, 3.55 sec" | "2 Std., 3.55 Sek." | "2 h, 3.55 s" | "2小时, "|
+| "3 days, 23 hr, 2 min" | "3 Tg., 23 Std., 2 Min." | "3 j, 23 h, 2 min" | "3天, 23小时, 2分钟" | 
+| "2 sec" | "2 Sek." | "2 s"| "2秒" |
+
+#### `DurationFormatter`
+
+We will export the class that is responsible for formatting the duration values so that we can use it for formatting values outside of the `Table` (a requirement in SLE).
+
+```ts
+export class DurationFormatter{
+    public constructor(private readonly lang: string) {}
+
+    // formats a value in the seconds domain into a localized display value that can 
+    // consist of days, hours, minutes, and seconds.
+    public format(value: TableNumberFieldValue): string {}
+}
+```
+
+### Angular integration
+
+An Angular directive will be created for the component. The component will not have form association, so a `ControlValueAccessor` will not be created.
+
+### Blazor integration
+
+A Blazor wrapper will be created for the component.
```

Comment:
Now that you've added `DurationFormatter` (thanks!), what framework integrations would we want for it?

I'm guessing for Angular we'd just re-export it from nimble-angular so that clients don't need to depend on nimble-components to access it?

For Blazor my gut is that it's not necessary initially but would be easy to add if a use case came up. 

Either way, let's document our proposals here.

And I ass